### DISCUSSION TITLE
UI - нелогичная позиция аккордеона

### DISCUSSION THREAD

**Discussion by kaavain** [cdek-it/widget#15] [Ideas]
И это уже писал - продублирую здесь.
Правый-верхний угол, кнопка аккордеон-меню открытия списка видимых на карте ПВЗ. Почему кнопка справа, а список открывается слева под поиском? Это, конечно, не критично от слова "совсем", но я считаю что надо:
В1: переместить кнопка налево к полю поиска , но тогда риск что юзер будет ее жать для выполнения поиска.
В2: открывать список где логично, под кнопкой. Тогда он будет закрывать кнопки масштаба и геопозиции, в чем я не вижу особой беды, хотя в теории юзер может захотеть при открытом списке изменить масштаб, чтобы увидеть обновленный список. Но думаю что, так как это можно сделать колесом мыши, то на таких извращенцев будет в разы меньше, чем испуганных внезапным появлением списка вдали от кнопки.

**Comment by vermorag** [comment]
Поскольку проблема не является критичной, обращений подобной тематики зарегистрировано мало, с 18.12.2023 по 10.01.2023 в компании СДЭК вводится запрет на выкладку новых версий модулей - данный запрос поставлен в план разработки на 2Q.2024. В случае увеличения количества обращений по данному вопросу - срок может быть пересмотрен.

**Comment by vermorag** [comment]
Исправлено в версии 3.8.0
